packages/client/src/providers/AuthProvider/index.js

```diff
@@ -1,18 +1,6 @@
 import React, { useState, useEffect, useContext, createContext } from 'react';
-import firebase from 'firebase/app';
-import 'firebase/auth';
 
-const firebaseConfig = {
-    apiKey: 'AIzaSyCSyjSnzA5mp8Y3ro904vg_wxMsG3ELsek',
-    authDomain: 'coved-test.firebaseapp.com',
-    databaseURL: 'https://coved-test.firebaseio.com',
-    projectId: 'coved-test',
-    storageBucket: 'coved-test.appspot.com',
-    messagingSenderId: '779651794617',
-    appId: '1:779651794617:web:8b9bf2f99882b58d337d68'
-};
-
-firebase.initializeApp(firebaseConfig);
+import { Auth, Mentor } from '../FirebaseProvider';
```

Comment:
what are your thoughts on making the collection related code live in another place, so for example

`import { Mentor } from '../models';`

Reply:
Sounds good.